borg/archiver.py

```diff
@@ -804,6 +801,8 @@ def __call__(self, parser, ns, values, option):
                                help="""toggle progress display while creating the archive, showing Original,
                                Compressed and Deduplicated sizes, followed by the Number of files seen
                                and the path being processed, default: %(default)s""")
+        subparser.add_argument('--changed', action='store_true', dest='changed', default=False,
+                               help="""display which files were added to the archive""")
```

Comment:
"log only added or modified files"
"log only files with content changes"
btw, how do you want to do that, just output A and M and nothing else?